_data/sites.json

```diff
@@ -9406,6 +9406,19 @@
         ]
     },
 
+    {
+        "name": "Hemingway Editor",
+        "url": "https://hemingwayapp.com/help#more-help",
+        "difficulty": "hard",
+        "notes": "Since there is no button, you must contact them through E-Mail — support@hemingwayapp.com. Thankfully, the wait time for their response is brief; however, this may differ from one to another.",
+        "notes_cz": "Protože není k dispozici žádné tlačítko, musíte je kontaktovat prostřednictvím e-mailu – support@hemingwayapp.com. Čekací doba na jejich odpověď je naštěstí krátká, může se však u každého lišit.",
+        "notes_sk": "Keďže tu nie je žiadne tlačidlo, musíte ich kontaktovať prostredníctvom e-mailu – support@hemingwayapp.com. Našťastie je doba čakania na ich odpoveď krátka, môže sa však líšiť od jednej osoby k druhej.",
```

Comment:
Can you please remove the email address from the notes, since there is already the "email" field with the information?